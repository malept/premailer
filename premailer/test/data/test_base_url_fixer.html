<html>
<head>
<title>Title</title>
</head>
<body>
<img src="/images/foo.jpg">
<img src="/images/bar.gif">
<img src="http://www.googe.com/photos/foo.jpg">
<a href="/home">Home</a>
<a href="http://www.peterbe.com">External</a>
<a href="subpage">Subpage</a>
<a href="#internal_link">Internal Link</a>
</body>
</html>
