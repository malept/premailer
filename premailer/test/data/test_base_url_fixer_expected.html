<html>
<head>
<title>Title</title>
</head>
<body>
<img src="http://kungfupeople.com/images/foo.jpg">
<img src="http://kungfupeople.com/images/bar.gif">
<img src="http://www.googe.com/photos/foo.jpg">
<a href="http://kungfupeople.com/home">Home</a>
<a href="http://www.peterbe.com">External</a>
<a href="http://kungfupeople.com/subpage">Subpage</a>
<a href="#internal_link">Internal Link</a>
</body>
</html>
